src/main/java/com/microsoft/sqlserver/jdbc/SQLServerParameterMetaData.java

```diff
@@ -424,6 +436,36 @@ private MetaInfo parseStatement(String sql) throws SQLServerException {
 
         return null;
     }
+    
+    private String removeCommentsInTheBeginning(String sql, int startCommentMarkCount, int endCommentMarkCount, String startMark, String endMark) {
```

Comment:
can you format the code with the [formatter](https://github.com/Microsoft/mssql-jdbc/blob/master/mssql-jdbc_formatter.xml)?